flink-connectors/flink-connector-jdbc/src/test/java/org/apache/flink/connector/jdbc/table/JdbcOutputFormatTest.java

```diff
@@ -91,181 +87,190 @@ public void tearDown() throws Exception {
-        try {
-            JdbcConnectorOptions jdbcOptions =
-                    JdbcConnectorOptions.builder()
-                            .setDriverName("org.apache.derby.jdbc.idontexist")
-                            .setDBUrl(DERBY_EBOOKSHOP_DB.getUrl())
-                            .setTableName(INPUT_TABLE)
-                            .build();
-            JdbcDmlOptions dmlOptions =
-                    JdbcDmlOptions.builder()
-                            .withTableName(jdbcOptions.getTableName())
-                            .withDialect(jdbcOptions.getDialect())
-                            .withFieldNames(fieldNames)
-                            .build();
-
-            outputFormat =
-                    new JdbcOutputFormatBuilder()
-                            .setJdbcOptions(jdbcOptions)
-                            .setFieldDataTypes(fieldDataTypes)
-                            .setJdbcDmlOptions(dmlOptions)
-                            .setJdbcExecutionOptions(JdbcExecutionOptions.builder().build())
-                            .build();
-            outputFormat.open(0, 1);
-            fail("Expected exception is not thrown.");
-        } catch (Exception e) {
-            assertTrue(findThrowable(e, IOException.class).isPresent());
-            assertTrue(findThrowableWithMessage(e, expectedMsg).isPresent());
-        }
+        assertThatThrownBy(
+                        () -> {
+                            JdbcConnectorOptions jdbcOptions =
+                                    JdbcConnectorOptions.builder()
+                                            .setDriverName("org.apache.derby.jdbc.idontexist")
+                                            .setDBUrl(DERBY_EBOOKSHOP_DB.getUrl())
+                                            .setTableName(INPUT_TABLE)
+                                            .build();
+                            JdbcDmlOptions dmlOptions =
+                                    JdbcDmlOptions.builder()
+                                            .withTableName(jdbcOptions.getTableName())
+                                            .withDialect(jdbcOptions.getDialect())
+                                            .withFieldNames(fieldNames)
+                                            .build();
+
+                            outputFormat =
+                                    new JdbcOutputFormatBuilder()
+                                            .setJdbcOptions(jdbcOptions)
+                                            .setFieldDataTypes(fieldDataTypes)
+                                            .setJdbcDmlOptions(dmlOptions)
+                                            .setJdbcExecutionOptions(
+                                                    JdbcExecutionOptions.builder().build())
+                                            .build();
+                            outputFormat.open(0, 1);
+                        })
+                .hasMessage(expectedMsg);
```

Comment:
We're not checking for the `IOException` here 🤔

Reply:
Good catch :)